ee/tabby-webserver/src/schema/auth.rs

```diff
@@ -297,6 +313,30 @@ pub struct RequestInvitationInput {
     pub email: String,
 }
 
+#[derive(Validate, GraphQLInputObject)]
+pub struct RequestPasswordResetInput {
+    #[validate(email(code = "email"))]
+    pub email: String,
+}
+
+#[derive(Validate, GraphQLInputObject)]
+pub struct PasswordResetInput {
+    #[validate(email(code = "email"))]
+    pub email: String,
+    pub code: String,
+    #[validate(length(
+        min = 8,
+        code = "password",
+        message = "Password must be at least 8 characters"
+    ))]
+    #[validate(length(
+        max = 20,
+        code = "password",
+        message = "Password must be at most 20 characters"
+    ))]
+    pub password: String,
```

Comment:
```suggestion
  pub code: String,
  pub password1: String,
  pub password2: String,
```

Reply:
Why no validations?

Reply:
Are you just saying to also add a `password2` field which ensures the two passwords are the same? Can that check not be performed on the frontend?

Reply:
> Why no validations?
I'll leave validations to your for implementation, just listing field for reference :)

> Are you just saying to also add a password2 field which ensures the two passwords are the same? Can that check not be performed on the frontend?
Yes, see [RegisterInput](https://github.com/TabbyML/tabby/blob/3db3d95eaedf7d347d035c1db87727cca46fe191/ee/tabby-webserver/src/service/auth.rs#L37) as reference. (in general, I suggest always following practices in components doing similar functionalities in repo)